pkg/start/start.go

```diff
@@ -489,11 +489,21 @@ type Context struct {
 	AutoUpdate              *autoupdate.Controller
 	StopOnFeatureGateChange *featuregates.ChangeStopper
 
-	CVInformerFactory                     externalversions.SharedInformerFactory
-	OpenshiftConfigInformerFactory        informers.SharedInformerFactory
-	OpenshiftConfigManagedInformerFactory informers.SharedInformerFactory
-	InformerFactory                       externalversions.SharedInformerFactory
-	OperatorInformerFactory               operatorexternalversions.SharedInformerFactory
+	// ClusterVersionInformerFactory should be used to get informers / listers for code that works with ClusterVersion resource
+	// singleton in the cluster.
+	ClusterVersionInformerFactory configinformers.SharedInformerFactory
+	// ConfigInformerFactory should be used to get informers / listers for code that works with resources from the
+	// config.openshift.io group, _except_ the ClusterVersion resource singleton.
```

Comment:
I did not understand why we use another factory for CV. But this is me, need more time to read the code.
My native guess is that their configurations are different for some reason.